packages/leaders-program/src/scss/components/_key-executive.scss

```diff
@@ -18,9 +18,4 @@
     font-weight: $leaders-key-executive-font-weight;
   }
 
```

Comment:
Actually, I believe the HTML markup can remain the same. 

Instead, change the CSS selector to use an `::after` on the `__name` only when both the `__name` and `__title` exist and are adjacent. I believe that would use the `+` operator.

Reply:
Although I think you can only access the second class name when doing adjacency... so perhaps that won't work. Is there any other CSS only solution... or a CSS+HTML solution that doesn't use so many if statements? Just trying to make it as concise as possible :)

Reply:
There isn't a way that I was able to find, but I can keep researching if needed.

Reply:
Nah, don't worry about it. I'll merge as-is :)